leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -30,26 +30,34 @@
 
 public final class SqlitePaxosStateLog<V extends Persistable & Versionable> implements PaxosStateLog<V> {
     private final Supplier<Connection> connectionSupplier;
+    private final String logNamespace;
 
-    private SqlitePaxosStateLog(Supplier<Connection> connectionSupplier) {
+    private SqlitePaxosStateLog(Supplier<Connection> connectionSupplier, String logNamespace) {
         this.connectionSupplier = connectionSupplier;
+        this.logNamespace = logNamespace;
     }
 
-    public static <V extends Persistable & Versionable> PaxosStateLog<V> createInitialized(Supplier<Connection> conn) {
-        SqlitePaxosStateLog<V> log = new SqlitePaxosStateLog<>(conn);
+    public static <V extends Persistable & Versionable> PaxosStateLog<V> createInitialized(
+            Supplier<Connection> conn,
```

Comment:
Can you create as many sqlite instances as you want if it's backed by the filesystem? Does it maintain some on-disk locks or something?